swarm64_run_tpc_benchmark

```diff
@@ -1,15 +1,16 @@
-#!/usr/bin/env python3
+#!/usr/bin/env python3.6
 
 
 import argparse
 import logging
 import sys
 
-from logging.config import fileConfig
+from os import chdir
 
+from logging.config import fileConfig
 from swarm64_tpc_toolkit.streams import Streams
 
-
+chdir(sys.path[0])
```

Comment:
That does not feel correct to me. It should then rather handle absolute paths properly.

Reply:
Do you have a suggestion ?

Reply:
For instance methods listed in https://stackoverflow.com/questions/3430372/how-do-i-get-the-full-path-of-the-current-files-directory to get the absolute path to the CWD and then use that for all remaining relative paths and turn them into absolute ones with `pathlib`.